library/find_unused_disk_module.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python
+
+DOCUMENTATION = '''
+---
+module: find_unused_disk_module
+short_description: Get first unused disk
```

Comment:
We probably need to return a list of unused disks rather than just one. Maybe return up to ten by default with an optional argument to set a max number of disks to return?

Reply:
Sounds good! I'll work on that